capstone/capdb/models.py

```diff
@@ -24,7 +24,8 @@
 from django.utils.encoding import force_bytes, force_str
 from django.core.files.base import ContentFile
 
-from capdb.storages import bulk_export_storage, case_image_storage, download_files_storage, writeable_download_files_storage, pdf_storage
```

Comment:
Seems like this file had a code formatter run on it? NBD, but since we don't currently have a formatter standardized for the project, I'd rather not run random formatters that generate a lot of arbitrary changes in the diff.

Reply:
Whoops— normally I only format the code I selected. I must have accidentally run it without code selected.